src/models/main-model.js

```diff
@@ -0,0 +1,22 @@
+import {Rx} from '@cycle/core';
+
+import makeVisibleIndices$ from './visible-indices-stream';
+
+function model(actions) {
+  let tableHeight$ = Rx.Observable.just(500);
+  let rowHeight$ = Rx.Observable.just(30);
+  let columns$ = Rx.Observable.just(['ID', 'ID * 10', 'Random Number']);
+  let rowCount$ = Rx.Observable.just(10000);
+  let scrollTop$ = actions.userScrolled$.startWith(0);
+  let visibleIndices$ = makeVisibleIndices$(
+    tableHeight$, rowHeight$, rowCount$, scrollTop$
+  );
+  let state$ = Rx.Observable.combineLatest(
+    tableHeight$, rowHeight$, columns$, rowCount$, visibleIndices$,
+    (tableHeight, rowHeight, columns, rowCount, visibleIndices) =>
+      ({tableHeight, rowHeight, columns, rowCount, visibleIndices})
+  );
+  return state$;
+}
+
+export default model;
```

Comment:
Well, more precisely this is what startWith does: http://rxmarbles.com/#startWith. It just returns you a **new** Observable (nothing Observer-related yet) where the first immediate event is the value you gave. The result is also a cold Observable. `Observable.just()` returns you a cold Observable as well. With regard to cold vs hot: https://egghead.io/lessons/rxjs-demystifying-cold-and-hot-observables-in-rxjs